uberenv.py

```diff
@@ -432,9 +444,11 @@ def patch(self):
         else:
             # let spack try to auto find compilers
             sexe("spack/bin/spack compiler find", echo=True)
-        dest_spack_pkgs = pjoin(spack_dir,"var","spack","repos","builtin","packages")
-        # hot-copy our packages into spack
-        sexe("cp -Rf {} {}".format(self.pkgs,dest_spack_pkgs))
+
+        if self.pkgs:
+            # hot-copy our packages into spack
```

Comment:
Suggestion/Nitpick: Maybe move the comment up one line and modify as follows:

```python
        # hot-copy our local packages into spack 
        if self.pkgs:
           dest_spack_pkgs = ...
```

Asking because I had to look up what ``self.pkgs`` contained.

Reply:
Done